www/src/templates/component-page-template.tsx

```diff
@@ -136,7 +138,10 @@ export default function PageTemplate({
             <p className="small mb-0">{mdx.frontmatter.notes}</p>
           </Alert>
         )}
-        <h1 className="mb-4">{mdx.frontmatter.title}</h1>
+        <div className="d-flex justify-content-between">
+          <h1 className="mb-4">{mdx.frontmatter.title}</h1>
+          <LeaveFeedback as={Hyperlink} />
```

Comment:
If we're using `Hyperlink` here to get it to have the external launch icon, we may also want to include it on the other in-page "Leave feedback" link towards the bottom.

Likewise, we could also use `as={Hyperlink}` in the docs site `footer` such that we get the correct `rel="noopener noreferrer"` attribute for security. That said, given none of the other external links in the footer include an external launch icon, we can use `Hyperlink`'s `showLaunchIcon={false}` to hide the icon for the footer only.